server/nuxeo-nxr-server/src/main/resources/templates/docker-json/lib/log4j2.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Log4j Configuration - For more configuration information and examples see https://logging.apache.org/log4j/2.x/ -->
+<Configuration monitorInterval="30">
+  <Appenders>
+    <!-- ============================== -->
+    <!-- Append messages to the console -->
+    <!-- ============================== -->
+    <Console name="ConsoleJSONAppender" target="SYSTEM_OUT">
+        <JsonLayout complete="false" compact="false"/>
```

Comment:
We should enable `compact` for a production configuration which is the intent of this file.

Furthemore, I think `eventEol` should be enable too in order to have one log per line.

`complete` looks like a debug option to me and shouldn't be changed:
> If true, the appender includes the JSON header and footer, and comma between records. Defaults to false.